commands/init/mod.ts

```diff
@@ -95,24 +97,30 @@ export const init = command("init", {
     );
 
     const existingLinks = await config.get("projects");
-    let app: paths["/projects/{id}"]["get"]["responses"]["200"]["content"][
+    let app: paths["/apps/{id}"]["get"]["responses"]["200"]["content"][
       "application/json"
     ];
     const link = existingLinks[dest];
 
     if (!link) {
+      const config = await find({});
+
       logger.info(`Project doesn't exist. Creating a new one.`);
-      const res = await loading(projects.create({ name: appName }), {
-        text: "Creating app",
-        enabled: !flags.json,
-      });
+
+      const res = await loading(
+        apps.create({ config }),
+        {
+          text: "Creating app",
+          enabled: !flags.json,
+        },
+      );
       asserts(res.ok, res);
       app = res.data;
     } else {
       logger.info(
         `Project already exists, skipping creation. (link: ${link.id})`,
       );
-      const res = await projects.get({ id: link.id });
+      const res = await apps.get({ id: link.id });
```

Comment:
IDK if this line or L133 is actually right anymore either because I think it'd have to be `id: app.projectId`. Linking is going to be super wonky now because when you get an app, you're not getting it by project ID (currently) you're getting it by the base deployment UUID.

Reply:
might need to make an API change to make this work